ops/internal/deployer/versions.json

```diff
@@ -4,5 +4,6 @@
   "op-contracts/v1.8.0": "op-deployer/v0.0.14",
   "op-contracts/v2.0.0": "op-deployer/v0.2.2",
   "op-contracts/v3.0.0": "op-deployer/v0.3.1",
-  "op-contracts/v4.0.0": "op-deployer/v0.4.0-rc.2"
+  "op-contracts/v4.0.0": "op-deployer/v0.4.0-rc.2",
+  "interop-rc-alpha": "f79ed8b9c9cbdbf8bb492074f3f98da7f072e21a"
```

Comment:
This is abusing the semantic meaning of this file somewhat. 

Other options:
1. Accept a list of additional git references (e.g. commits) to the `build-binaries.sh` script instead of declaring them here
2. Require devnets to always be deployed from a tagged version of op-deployer

Reply:
I think if we need to support any commit, we should change the overall format of this file or add an additional file. The mixing of `op-contracts version` and `devnet name` --> `op-deployer version` is awkward

Reply:
Have reverted this change.